documentation/staging/content/faq/external-clients.md

```diff
@@ -23,6 +23,8 @@ description: "This FAQ describes approaches for giving WebLogic applications acc
 - [Enabling unknown host access](#enabling-unknown-host-access)
    - [When is it necessary to enable unknown host access?](#when-is-it-necessary-to-enable-unknown-host-access)
    - [How to enable unknown host access](#how-to-enable-unknown-host-access)
+- [Configuring WebLogic Server Affinity Load Balancing Algorithm](#configuring-weblogic-server-affinity-load-balancing-algorithm)
+- [Configuring External Listen Address for WebLogic Default Channel](#configuring-external-listen-address-for-weblogic-default-channel) 
```

Comment:
I just had a customer ask how to put together a cross-namespace URL for the stand-alone server case.

Please update 
```
- The URL used by the application can begin with `t3` or `t3s`, and the  DNS address in the URL must be fully specified to differentiate the namespace. ...
```
to (a) use 'svc.cluster.local' or drop it all-together, and (b) to specifically mention the stand-alone server case.

Something like:
```
- The URL used by the application can begin with `t3` or `t3s`, and the  DNS address in the URL must be fully specified to differentiate the namespace.  For example:
   - If the target resource is in WebLogic cluster `mycluster` with    domain UID `myuid` in namespace `myns`, and the cluster is    listening on port `8001`, then the cluster service name     that is generated by the operator will    be `myuid-cluster-mycluster` and the full URL will be `t3://myuid-cluster-mycluster.myns:8001`.
   - If the target resource is on stand-alone WebLogic server `myserver` with domain UID `myuid` in namespace `myns`, and the server is   listening on port `7001`, then the server service name   that is generated by the operator will be `myuid-myserver` and the full URL will be `t3://myuid-myserver.myns:8001`.
```